Calculated elements are not searchable by default (#343)
cap-js/docs: guides/providing-services.md

#### Using the `@cds.search` Annotation {#using-cds-search-annotation}

> By default all concrete elements of type `String` of an entity are searched. Yet, sometimes you may want to deviate from this default and specify a different set of searchable elements, or to extend the search to associated entities. Use the `@cds.search` annotation to do so. The general usage is:

Reviewer: ```suggestion
By default only elements of type `String` of an entity are searched, provided they are not [calculated](../cds/cdl#calculated-elements) or [virtual](../cds/cdl#virtual-elements). Yet, sometimes you may want to deviate from this default and specify a different set of searchable elements, or to extend the search to associated entities. Use the `@cds.search` annotation to do so. The general usage is:
```

Searches all elements of type `String` excluding the element `isbn`, which leaves the `title` and `descr` elements to be searched.

> You can extend the search to calculated elements which are not searchable by default.

Reviewer: ```suggestion
You can extend the search to calculated elements, which are not searchable by default.
```
